Update custom email guide w/ new code editor (#3586)
okta/okta-developer-docs: packages/@okta/vuepress-site/docs/guides/custom-email/main/index.md

> **Note:** To access email customization with a free developer edition of an Okta org, you need to contact [Okta support](https://support.okta.com/help).

1. In the Admin Console, go to **Customizations** > **Branding**.
1. In the **Communications** section, in the **Emails** box, click **Edit**.
1. In the **Email Templates** table, click a template name.
   - The code editor is in preview mode.
   - **Audience**: Select the email audience.

>       - All users

Reviewer: should these items be bold, since they are selecting one? And do they need to be bulleted? You could just say:

Select the email audience: All users, Admin only, or Disabled for all.

Author: Good point. Thank you :)

      - Admin only
      - Disabled for all
   - **Translations**: Customize email template translations. See [Add translations](#add-translations).
   - **Language**: Select the default language for the email template. See [Add translations](#add-translations).

>    - **Subject**: The subject line for the template in the default language. You can edit the subject line when the template is in read-write mode.

Reviewer: ```suggestion
   - **Subject**: The subject line for the template in the default language. You can edit the subject line when the template is in read/write mode.
```

>    - In the preview window, to see a translation of a customized template, select a language from the drop-down menu. See [Add translations](#add-translations).

Reviewer: ```suggestion
   - In the preview window, to see a translation of a customized template, select a language from the dropdown menu. See [Add translations](#add-translations).
```

> 1. Click **Edit** to put the editor in read-write mode.

Reviewer: ```suggestion
1. Click **Edit** to put the editor in read/write mode.
```

1. Make changes directly in the editor. If you type `$`, `#` or `{`, the editor provides a list of available variables that you can use. See [Use Velocity Templating Language (VTL)](#use-velocity-templating-language).

>    - Click the full page button to see the code editor in full page mode.

Reviewer: ok, I googled this and I'm wrong. Let's just go with full-screen as it's the common use. My bad.
```suggestion
   - Click the full-screen button to see the code editor in full-screen mode.
```

>    - At the top of the code editor, click the language to see a dropdown list. Select a language from the list of active template translations. See [Add translations](#add-translations).

Author: good point. It may also work in the step 3 section as something they change _before_ they start editing.

## Add translations

**Note:** Once a template is customized, other languages are only included when they are customized. If all customized translations are removed, then the 27 default translations are used.

> The default language is used when the end user’s locale does not match any email customizations. You can designate any added translation as the default language by selecting it from the **Default Language** dropdown box. You can edit the templates through the pencil icon, but you can't delete the default language template.

Author: Addressing now.

## Add translations

> **Note:** Once a template is customized, other languages are only included when they are customized. If all customized translations are reset, then the 27 default translations are used.

Reviewer: ```suggestion
**Note:** After a template is customized, other languages are only included when they are customized. If all customized translations are reset, then the 27 default translations are used.
```

> **Note:** Once a template is customized, other languages are only included when they are customized. If all customized translations are removed, then the 27 default translations are used.

Reviewer 2: If all customized translations are `reset` ?

> **Note:** Once a template is customized, other languages are only included when they are customized. If all customized translations are removed, then the 27 default translations are used.

Author: ```suggestion
**Note:** Once a template is customized, other languages are only included when they are customized. If all customized translations are reset, then the 27 default translations are used.
```

> The default language is used when the end user’s locale does not match any email customizations. You can edit the templates through the pencil icon, but you can't delete the default language template.

Reviewer: ```suggestion
The default language is used when the end user’s locale doesn't match any email customizations. You can edit the templates through the pencil icon, but you can't delete the default language template.
```

1. In the Admin Console, go to **Customizations** > **Branding**.
1. In the **Communications** section, in the **Emails** box, click **Edit**.
1. In the **Email Templates** box, select an email template that you customized.
   - The code editor is in preview mode and shows the default language version.
   - **Translations**: Customize email template translations:
      1. Click **Edit**.

>       1. Under under **Inactive Translations**, next to the language you want to customize, click **Customize**.

Reviewer: ```suggestion
      1. Under **Inactive Translations**, next to the language that you want to customize, click **Customize**.
```
